tsdb/head.go

```diff
@@ -1237,6 +1347,13 @@ func (a *headAppender) log() error {
 			return errors.Wrap(err, "log series")
 		}
 	}
+	if len(a.exemplars) > 0 {
+		rec = enc.Exemplars(a.exemplars, buf)
+		buf = rec[:0]
+		if err := a.head.wal.Log(rec); err != nil {
+			return errors.Wrap(err, "log exemplars")
+		}
+	}
```

Comment:
I thought this would be done in the next PR and this PR is only about the in-memory queue. What am I missing?